fr/lecons/nettoyer-ses-donnees-avec-openrefine.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Nettoyer ses données avec OpenRefine
+layout: lesson
+date: 2013-08-05
+translation_date: 2018-11-27
+authors:
+- Seth van Hooland
+- Ruben Verborgh
+- Max De Wilde
+reviewers:
+- Patrick Burns
+- Nora McGregor
+editors:
+- Adam Crymble
+translator:
+- Sybille Clochet
+translation-editor:
+- Sofia Papastamkou
+translation-reviewer:
+- François Dominic Laramée
+- Marie Puren
+difficulty: 2
+exclude_from_check:
+  - review-ticket
+activity: transforming
+topics: [data-manipulation]
+abstract: "Ce tutoriel explique comment les chercheur(e)s peuvent déterminer l'état de précision de leurs données et comment agir pour l'améliorer." 
+original: /en/lessons/cleaning-data-with-openrefine
```

Comment:
Please note in the [translation guidelines](https://programminghistorian.org/fr/consignes-redacteurs#3-ajouter-les-m%C3%A9tadonn%C3%A9es-en-yaml-dans-le-fichier-de-la-le%C3%A7on) that `original` should only show the lesson slug, no other path elements, e.g.

```
original: cleaning-data-with-openrefine
```